src/test/java/reposense/util/FileUtilTest.java

```diff
@@ -5,22 +5,52 @@
 import java.nio.file.Files;
 import java.nio.file.Path;
 import java.nio.file.Paths;
+import java.util.Arrays;
+import java.util.HashSet;
 
 import org.junit.After;
 import org.junit.Assert;
 import org.junit.Test;
 
+import reposense.report.SummaryReportJson;
+
 public class FileUtilTest {
 
     private static final Path FILE_UTIL_TEST_DIRECTORY = new File(FileUtilTest.class.getClassLoader()
             .getResource("FileUtilTest").getFile()).toPath().toAbsolutePath();
+    private static final Path RELEVANT_REPO_DIRECTORY_PATH = Paths.get(FILE_UTIL_TEST_DIRECTORY.toString(),
+            "reposense-report-test");
     private static final Path ARCHIVE_ZIP_PATH = Paths.get(FILE_UTIL_TEST_DIRECTORY.toString(),
             FileUtil.ZIP_FILE);
     private static final Path EXPECTED_UNZIPPED_DIRECTORY_PATH = Paths.get(FILE_UTIL_TEST_DIRECTORY.toString(),
             "expectedUnzip");
     private static final Path TEST_ZIP_PATH = Paths.get(FILE_UTIL_TEST_DIRECTORY.toString(), "testZip.zip");
     private static final Path UNZIPPED_DIRECTORY_PATH = Paths.get(FILE_UTIL_TEST_DIRECTORY.toString(),
             "UnzippedFolder");
+    private static final Path[] REPO_FOLDERS = {Paths.get("reposense_testrepo-Beta"),
+            Paths.get("reposense_testrepo-Charlie")};
+    private static final Path EXPECTED_RELEVANT_FOLDER_PATH = Paths.get(FILE_UTIL_TEST_DIRECTORY.toString(),
+            "expectedRelevantUnzippedFiles");
+
+    /**
+     * Ensures that only the specified files and folders with the corresponding file types get zipped.
+     */
+    @Test
+    public void zip_onlyRelevantFiles_success() throws IOException {
+        HashSet<Path> relevantFolders = new HashSet<>(Arrays.asList(REPO_FOLDERS));
+        HashSet<Path> relevantFiles = new HashSet<>();
+        relevantFolders.add(Paths.get(RELEVANT_REPO_DIRECTORY_PATH + File.separator
```

Comment:
The `relevantFolders` seems already have the addresses of these two folders?

Reply:
See
```
    private static final Path[] REPO_FOLDERS = {Paths.get("reposense_testrepo-Beta"),
            Paths.get("reposense_testrepo-Charlie")};
    HashSet<Path> relevantFolders = new HashSet<>(Arrays.asList(REPO_FOLDERS));
```

Reply:
My bad; forgot to remove those